mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionServiceImpl.java

```diff
@@ -201,6 +202,11 @@ public Future<JobExecution> setJobProfileToJobExecution(String jobExecutionId, J
         }));
   }
 
+  @Override
+  public Future<JobProfileInfoCollection> getRelatedJobProfiles(int offset, int limit, String tenantId) {
+    return jobExecutionDao.getRelatedJobProfiles(offset, limit, tenantId);
+  }
+
```

Comment:
This endpoint necessary for building dropdown with all possible job profiles to be able to filter by them.
Not sure that UI team will do paginated requests to fetch all job profiles. Thinking that we don't need pagination here

Reply:
Hi @SerhiiNosko. It was specified in the ticket. If reporter(Ruslan Lavrov) is agree with you then I can remove offset param.